contract.js

```diff
@@ -102,7 +104,8 @@ const initialize = () => {
 
   const onClickConnect = async () => {
     try {
-      await ethereum.enable()
+      accounts = await ethereum.send('eth_requestAccounts')
+      handleNewAccounts(accounts)
```

Comment:
Nit: this is assigning the result to `accounts` here, then assigning it again in `handleNewAccounts`

Maybe it should be a local variable instead? e.g.

```suggestion
      const newAccounts = await ethereum.send('eth_requestAccounts')
      handleNewAccounts(newAccounts)
```